node/network/statement-distribution/src/lib.rs

```diff
@@ -1,6 +1,3 @@
-// Copyright 2020 Parity Technologies (UK) Ltd.
-// This file is part of Polkadot.
-
```

Comment:
probably unintended

Reply:
Yeah, thanks - that was some auto merge, I guess.